kakao/src/main/kotlin/com/agoda/kakao/core/Configuration.kt

```diff
@@ -0,0 +1,10 @@
+package com.agoda.kakao.core
+
+import com.agoda.kakao.common.interactors.Interactor
+
+class Configuration{
+    val interactor = Interactor()
+    fun onInteraction(onInteraction: Interactor.() -> Unit){
```

Comment:
`onViewInteraction`, since we do stuff not only with `ViewInteraction`, but with `DataInteraction` and `WebInteraction` as well